generate-release/src/migration_guide.rs

```diff
@@ -85,10 +102,7 @@ As a result, the Minimum Supported Rust Version (MSRV) is "the latest stable rel
     }
     writeln!(&mut output, "</div>")?;
 
-    println!(
-        "\nFound {} breaking PRs merged by bors",
-        merged_breaking_prs.len()
-    );
+    println!("\nFound {} breaking PRs merged by bors", count);

```

Comment:
```suggestion
    println!("\nFound {} breaking PRs merged", count);

```

Reply:
Oh, nice catch, I didn't even notice, that's a flash from the past 😅